m4/tlog/conf_cmd.m4

```diff
@@ -310,7 +310,7 @@ m4_dnl
 m4_define(
     `M4_CONF_CMD_HELP_OPTS',
     `m4_pushdef(`m4_orig_divnum', m4_divnum)m4_divert(-1)
-        m4_pushdef(`M4_CONTAINER', m4_defn(`M4_CONF_CMD_HELP_OPTS_CONTAINER')), 
+        m4_pushdef(`M4_CONTAINER', m4_defn(`M4_CONF_CMD_HELP_OPTS_CONTAINER')),
```

Comment:
Could you please make this a separate change, or not change it at all?

Reply:
Oh It is weird. I do not change anything in this line.